orquesta/conducting.py

```diff
@@ -184,7 +184,8 @@ def get_staged_tasks(self, filtered=True):
         if not filtered:
             return self.staged
 
-        return [x for x in self.staged if x["ready"] and not x.get("completed", False)]
+        resp = [x for x in self.staged if x["ready"] and not x.get("completed", False)]
```

Comment:
this change is not needed, correct?

Reply:
correct, I'll revert some of the additional changes that were made (just moved variables out of return statements to get timing on them, didn't move back)